heudiconv/heuristics/reproin.py

```diff
@@ -643,8 +681,8 @@ def from_series_info(name: str) -> Optional[str]:
             "_Scout" in curr_seqinfo.series_description
             or (
                 datatype == "anat"
-                and datatype_suffix
-                and datatype_suffix.startswith("scout")
+                and filename_suffix_parts[-1]
+                and filename_suffix_parts[-1].startswith("scout")
```

Comment:
what does this have to do with phase suffix? nothing in short commit messages hints on possible relation to this change... why is it?